packages/aws-lc-fips/aws-lc-fips.spec

```diff
@@ -0,0 +1,77 @@
+Name: %{_cross_os}aws-lc-fips
+Version: 3.0.0
+Release: 1%{?dist}
+Summary: AWS-LC cryptographic library (FIPS)
+License: Apache-2.0 OR ISC OR BSD-3-Clause OR MIT OR CC0-1.0 OR OpenSSL OR SSLeay-standalone
+URL: https://github.com/aws/aws-lc
+
+Source0: https://github.com/aws/aws-lc/archive/AWS-LC-FIPS-%{version}/aws-lc-AWS-LC-FIPS-%{version}.tar.gz
+
+BuildRequires: %{_cross_os}glibc-devel
+
+Requires: %{_cross_os}glibc
+
+%description
+AWS-LC is a general-purpose cryptographic library maintained by the
+AWS Cryptography team for AWS and their customers. It іs based on code
+from the Google BoringSSL project and the OpenSSL project. This version
+includes FIPS support.
+
+%package devel
+Summary: Development files for %{name}
+Requires: %{name}%{?_isa} = %{version}-%{release}
+
+%description devel
+AWS-LC development files from package %{name}.
+
+%prep
+%setup -n aws-lc-AWS-LC-FIPS-%{version}
+
+%build
+%set_cross_build_flags
+
+mkdir -p aws-lc-build
+cd aws-lc-build
+
+%cross_cmake ../ \
+    -GNinja \
+    -DCMAKE_BUILD_TYPE=RelWithDebInfo \
+    -DBUILD_SHARED_LIBS=ON \
```

Comment:
If nothing needs `libssl.so`, only `libcrypto.so`, I'd prefer to disable it to cut down on surface area for potential vulnerabilities:
```suggestion
    -DBUILD_SHARED_LIBS=ON \
    -DBUILD_LIBSSL=OFF \
```

Reply:
Okay, I tried building now and it does activate a fair few things in systemd (to have openssl available) - I spotted that udev did not build, neither did the shared library.
I do not really have time to dive into this and craft a patch to only enable the crypto dependencies in Systemd (and not the libssl ones), but I do apologise for polluting your otherwise very pristine distro with libssl. :smiling_face_with_tear: